http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -144,10 +154,21 @@ object KubernetesServiceDbQueries {
           _ <- nodepoolQuery.updateStatus(createCluster.nodepoolId, NodepoolStatus.Provisioning)
         } yield ()
       )
-      _ <- nodepoolQuery.updateStatus(nodepoolId, NodepoolStatus.Provisioning)
+      _ <- nodepoolId.traverse(id => nodepoolQuery.updateStatus(id, NodepoolStatus.Provisioning))
       _ <- appQuery.updateStatus(appId, AppStatus.Provisioning)
     } yield ()
 
+  def markPendingCreating(clusterId: KubernetesClusterLeoId, nodepoolIds: List[NodepoolLeoId])(
```

Comment:
This is for the batch create, right? Could we maybe name the method differently instead of overloading `markPendingCreating`?